src/shared/containers/ResourceEditor.tsx

```diff
@@ -69,7 +66,7 @@ const ResourceEditorContainer: React.FunctionComponent<{
     return await nexus.Resource.getSource(
       orgLabel,
       projectLabel,
-      resourceId,
+      encodeURIComponent(resourceId),
```

Comment:
This is behaving a bit weird, similar error when I click 'Metadata' or 'Expand' buttons.